### PR TITLE
code cleanup and style formatting

### PR DESCRIPTION
**Pull request by RubensCordeiroo**
I did a cleaning and there were many changes, just format it. If you want to merge this PR, otherwise I delete it.  Your work is amazing. I am very happy to be able to contribute to the best library to manipulate images. Thanks a lot

### CODE REVIEW

File: tests/image-magick/read.spec.ts [hunk 1]
```diff
@@ -46,8 +46,8 @@ describe('ImageMagick#read', () => {
                 await bogusAsyncMethod();
             });
         })
-        .rejects
-        .toThrowError('ReadPNGImage');
+            .rejects
```
Comment (dlemstra) [inline tests/image-magick/read.spec.ts:49]: This also looks indented too much?

File: tests/image-magick/read-collection.spec.ts [hunk 2]
```diff
@@ -53,8 +53,8 @@ describe('ImageMagick#readCollection', () => {
                 await bogusAsyncMethod();
             });
         })
-        .rejects
-        .toThrowError('ReadPNGImage');
+            .rejects
```
Comment (dlemstra) [inline tests/image-magick/read-collection.spec.ts:56]: This looks indented too much?

**Review by dlemstra — COMMENTED** [review]
Thanks for all these changes and adding some extra compared to your earlier PR. Found two spots where the change looks a bit strange.

File: tests/image-magick/read.spec.ts (hunk 1 above)
Comment (RubensCordeiroo) [reply tests/image-magick/read.spec.ts:49]: I'm going to fix it.

Comment (RubensCordeiroo) [reply tests/image-magick/read.spec.ts]: commit ac5a0b4d78665beb494217032bd8a71722dfe5c0.

File: tests/image-magick/read-collection.spec.ts (hunk 2 above)
Comment (RubensCordeiroo) [reply tests/image-magick/read-collection.spec.ts]: commit ac5a0b4d78665beb494217032bd8a71722dfe5c0.
Thanks to let me know

**Review by dlemstra — APPROVED** [review]
Thanks for fixing this.

**Pull request merged**